From eaaf3c3f354aaa7febde9ec6ecd4905c684ce23c Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Wed, 6 Aug 2014 03:45:55 -0400
Subject: [acpi] numa: Use __weak, not the gcc-specific version

Message-id: <20140806034555.1438.64369.stgit@gir.stowe>
Patchwork-id: 87426
O-Subject: [RHEL7.1 PATCH 17/69] ACPI / numa: Use __weak, not the gcc-specific version
Bugzilla: 1110896
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110896
Upstream Status: beffbe54fa2df3a80d74f168c38e0248adbde082

commit beffbe54fa2df3a80d74f168c38e0248adbde082
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Mon Jan 27 17:19:15 2014 -0700

    ACPI / numa: Use __weak, not the gcc-specific version

    Use "__weak" instead of the gcc-specific "__attribute__ ((weak))".

    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
    Acked-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/acpi/numa.c b/drivers/acpi/numa.c
index 7d83ab8..0f01486 100644
--- a/drivers/acpi/numa.c
+++ b/drivers/acpi/numa.c
@@ -194,7 +194,7 @@ static int __init acpi_parse_slit(struct acpi_table_header *table)
  return 0;
 }
 
-void __init __attribute__ ((weak))
+void __init __weak
 acpi_numa_x2apic_affinity_init(struct acpi_srat_x2apic_cpu_affinity *pa)
 {
  printk(KERN_WARNING PREFIX
-- 
1.7.1